### PR TITLE
Fix HTML errors and warnings

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR fixes some HTML errors and warnings:
* remove `title` tag, because `jekyll-seo-tag` creates another one
* add `lang` attribute to `html` tag
* remove unneeded `type` attribute on `script` tags
* add a needed alt attribute to `img` tags

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,8 @@
 layout: null
```
Comment (lpugin) [inline index.html:2]: You need to add `title: Verovio` for seo

Comment (rettinghaus) [reply index.html:2]: You mean in the `_config.yml`?

Comment (rettinghaus) [reply index.html:2]: Nevermind.

**Comment by rettinghaus** [comment]
That looks promising: https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.verovio.org%2Findex.html

**Pull request merged**